template/v1/types.go

```diff
@@ -111,6 +111,7 @@ type TemplateInstance struct {
 	Spec TemplateInstanceSpec `json:"spec" protobuf:"bytes,2,opt,name=spec"`
 
 	// status describes the current state of this TemplateInstance.
+	// +optional
```

Comment:
do these have any effect, e.g. on the openshift-apiserver openapi spec?

Reply:
Yes, they should not be required, and that's one of the reasons https://github.com/openshift/oc/pull/149 fails.